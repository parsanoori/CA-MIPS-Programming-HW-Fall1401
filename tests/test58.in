4
877345822
522260574
751596509
74912648