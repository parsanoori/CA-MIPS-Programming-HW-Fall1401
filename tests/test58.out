74912648
522260574
751596509
877345822